src/app/layouts/base-layout/components/header/header.tsx

```diff
@@ -29,12 +29,27 @@ const navLinks = [
     href: ROUTES.COMMUNITY,
     dropdownInner: <SchoolMenu heading="community" color="dark" hasTitle={false} />,
   },
+  {
+    label: 'Mentorship',
+    href: ROUTES.MENTORSHIP,
+    dropdownInner: <SchoolMenu heading="mentorship" color="dark" hasTitle={false} />,
+  },
 ];
 
 export const Header = () => {
   const [isMenuOpen, setMenuOpen] = useState(false);
-  const [color, setColor] = useState('gray');
   const { key, hash, pathname } = useLocation();
+
+  useEffect(() => {
+    if (pathname.includes(ROUTES.MENTORSHIP)) {
+      setColorName('blue');
+    } else {
+      setColorName('gray');
+    }
+  }, [pathname]);
+  const [colorName, setColorName] = useState('white');
+  const [color, setColor] = useState(colorName);
```

Comment:
Is there a real need to use two states for this? Because it's so confusing when looking at two variables with nearly the same name

Reply:
It's a decision I don't like myself. But I don't have another working solution yet.
I will be glad if you can tell me how to make the color change:
- blue for `mentorship` and `mentorship/course`
- yellow for all other pages

Reply:
Thanks for your help! It works.